src/app/core/services/kafka/kafka.service.ts

```diff
@@ -172,10 +181,13 @@ export class KafkaService {
       .then(cache => Object.keys(cache).reduce((s, k) => (k ? s + 1 : s), 0))
   }
 
-  sendKafkaEvent(type, cacheValue, error?) {
-    this.firebaseAnalytics.logEvent(type, {
-      name: cacheValue.name,
-      questionnaire_timestamp: String(cacheValue.kafkaObject.time),
+  sendDataEvent(type, cacheValue: CacheValue, error?) {
+    const value = cacheValue.kafkaObject.value
+    this.analytics.logEvent(type, {
+      name: cacheValue.repository ? SchemaType.ASSESSMENT : cacheValue.name,
+      timestamp: String(value.time),
```

Comment:
What is this used for? We should try to keep the parameters to a minimum as there is a limit on the firebase dashboard. I guess the time an event was reported should be enough info (we can possibly remove `timestamp` if not necessary)

Reply:
`timestamp` is the timestamp of when non-questionnaire data was sent like `completion-log` or `timezone`. I separated it because `completion-log` has `timestamp` and `timeNotification` = `questionnaire_timestamp` in this case. But `timezone` only has `timestamp`. And questionnaires has `timestamp` = `timeCompleted` and `questionnaire_timestamp` = `timeNotification`.  I guess it was just to be as detailed as possible? But I'm not sure how detailed it should be?